cmd/catalog.go

```diff
@@ -21,15 +22,26 @@ var catalogCmd = &cobra.Command{
 
 var catalogListCmd = &cobra.Command{
 	Use:   "list",
+	Short: "List all available component & service types.",
+	Long:  "List all available component and service types from OpenShift",
+	Example: `  # Get the supported components
+  odo catalog list components
+
+  # Get the supported services from service catalog
+  odo catalog list services
+`,
+}
+
+var catalogListComponentCmd = &cobra.Command{
+	Use:   "components",
 	Short: "List all available component types.",
 	Long:  "List all available component types from OpenShift's Image Builder.",
 	Example: `  # Get the supported components
-  odo catalog list
+  odo catalog list components
 
   # Search for a supported component
-  odo search nodejs
+  odo catalog search nodejs
```

Comment:
shouldn't the command be `odo catalog search components` ? How Am I going to define if I'm searching for component or service?